tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -322,12 +337,56 @@ private void setUpWorkflowApi() {
         defaultApiClient.setBasePath(this.workflowRunnerConfig.getServerUrl());
         setWorkflowsApi(new WorkflowsApi(defaultApiClient));
     }
-    private void runToolTesterOnWorkflows(String pathToConfigFile) throws InterruptedException {
+
+    private List<File> getAllFilesInDirectory(File directory) {
+        List<File> filesInDirectory = new ArrayList<>();
+        for (File file: directory.listFiles()) {
+            if (file.isDirectory()) {
+                filesInDirectory.addAll(getAllFilesInDirectory(file));
+            } else {
+                filesInDirectory.add(file);
+            }
+        }
+        return filesInDirectory;
+    }
+
+    private void uploadResults(String urlToUploadTo, String resultsDirectoryString, String pathToConfigFile) {
+        this.workflowRunnerConfig = new WorkflowRunnerConfig(pathToConfigFile);
+        setUpExtendedGa4GhApi(urlToUploadTo);
+        Gson gson = new Gson();
```

Comment:
Should make this a constant so it's only instantiated once

Reply:
Fixed